recipes/rank_filter/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "rank_filter" %}
+{% set version = "0.4.11" %}
+{% set checksum = "246f5c73ebd03314d9b2cff3ede74330" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ checksum }}
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - boost >=1.56
+    - python
+    - setuptools >=18.0
+    - numpy x.x
+    - numpy >=1.7
+    - cython >=0.23
+
+  run:
+    - boost >=1.56
+    - python
+    - numpy x.x
+    - numpy >=1.7
```

Comment:
Technically this isn't needed - it can only be a numpy which was a suitable build version (which is >=1.7 anyway).